Makefile

```diff
@@ -1048,9 +1048,13 @@ LINT_MD_DOC_FILES = $(shell ls doc/**/*.md)
 run-lint-doc-md = tools/remark-cli/cli.js -q -f $(LINT_MD_DOC_FILES)
 # Lint all changed markdown files under doc/
 tools/.docmdlintstamp: $(LINT_MD_DOC_FILES)
+ifeq ("$(HAS_CRYPTO)", "true")
```

Comment:
Slightly more idiomatic: `ifeq ($(HAS_CRYPTO),true)`

Reply:
I'll update this shortly.